compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -1919,6 +1939,8 @@ extern "C" {
         kind: DebugEmissionKind,
         DWOId: u64,
         SplitDebugInlining: bool,
+        DebugInfoForProfiling: bool,
```

Comment:
There's a signature mismatch between Rust and C++ versions of `LLVMRustDIBuilderCreateCompileUnit`, this parameter needs to be removed.

Reply:
the C++ function has a `DebugInfoForProfiling` arg [before](https://llvm.org/doxygen/classllvm_1_1DIBuilder.html#a36f7af99f64b66e158b210e4433a1aed) the `DebugNameTableKind`, so I mirrored it here. Or do you mean just past the default `false` value to the C++ function inside the Rust ffi encapsulation?

Reply:
I mean this function (`LLVMRustDIBuilderCreateCompileUnit`) has the `DebugInfoForProfiling` parameter here, but not in `compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp`.